src/scss/components/_media-element.scss

```diff
@@ -10,6 +10,7 @@
 $mediaElement-img-width         : 50px;
 $mediaElement-img-width--mid    : 78px;
 $mediaElement-img-width--wide   : 67px;
+$mediaElement-img-borderRadius  : spacing() !default;
```

Comment:
Do you definitely want this borderRadius hooked into the spacing?  Might be better to change it so they aren't linked as if the spacing changes, I'm guessing you wouldn't want/expect this to update?

Reply:
Makes sense, I saw `8px` and instantly jumped to the `spacing` helper!